### PR TITLE
Update upstash-redis.md

### PR DESCRIPTION
**Pull request by Itsnotaka**
## ☕️ Reasoning

What changes are being made? What feature/bug is being fixed here?
The doc was outdated and the code written within the docs are incorrect at the time of writing this.

## 🧢 Checklist

- [x] Documentation
- [ ] Tests
- [ ] Ready to be merged

## 📌 Resources

- [Contributing guidelines](./CONTRIBUTING.md)
- [Code of conduct](./CODE_OF_CONDUCT.md)
- [Contributing to Open Source](https://kcd.im/pull-request)

### CODE REVIEW

File: docs/docs/adapters/upstash-redis.md [hunk 1]
```diff
@@ -17,12 +17,12 @@ Configure your NextAuth.js to use the Upstash Redis Adapter:
...
+import { Redis } from "@upstash/redis";
 
-const redis = upstashRedisClient(
-  process.env.UPSTASH_REDIS_URL,
-  process.env.UPSTASH_REDIS_TOKEN
-)
+const redis =const redis = new Redis({
+  url:  process.env.UPSTASH_REDIS_URL,
+  token:  process.env.UPSTASH_REDIS_TOKEN
+});
```
Comment (lluia) [inline docs/docs/adapters/upstash-redis.md:25]:
```suggestion
const redis = new Redis({
  url:  process.env.UPSTASH_REDIS_URL,
  token:  process.env.UPSTASH_REDIS_TOKEN
});
```

**Review by 0ubbe — CHANGES_REQUESTED** [review]
Thanks 💚

Small formatting issue.

File: docs/docs/adapters/upstash-redis.md [hunk 2]
```diff
@@ -17,12 +17,12 @@ Configure your NextAuth.js to use the Upstash Redis Adapter:
 import NextAuth from "next-auth"
 import GoogleProvider from "next-auth/providers/google"
 import { UpstashRedisAdapter } from "@next-auth/upstash-redis-adapter"
-import upstashRedisClient from "@upstash/redis"
+import { Redis } from "@upstash/redis";
```
Comment (balazsorban44) [inline docs/docs/adapters/upstash-redis.md:20]:
```suggestion
import { Redis } from "@upstash/redis"
```

File: docs/docs/adapters/upstash-redis.md [hunk 3]
```diff
@@ -17,12 +17,12 @@ Configure your NextAuth.js to use the Upstash Redis Adapter:
...
 
-const redis = upstashRedisClient(
-  process.env.UPSTASH_REDIS_URL,
-  process.env.UPSTASH_REDIS_TOKEN
-)
+const redis = new Redis({
+  url:  process.env.UPSTASH_REDIS_URL,
+  token:  process.env.UPSTASH_REDIS_TOKEN
+});
```
Comment (balazsorban44) [inline docs/docs/adapters/upstash-redis.md:25]:
```suggestion
})
```

File: docs/docs/adapters/upstash-redis.md [hunk 4]
```diff
@@ -17,12 +17,12 @@ Configure your NextAuth.js to use the Upstash Redis Adapter:
...
-import upstashRedisClient from "@upstash/redis"
+import { Redis } from "@upstash/redis"
 
-const redis = upstashRedisClient(
-  process.env.UPSTASH_REDIS_URL,
-  process.env.UPSTASH_REDIS_TOKEN
-)
+const redis = new Redis({
+  url:  process.env.UPSTASH_REDIS_URL,
+  token:  process.env.UPSTASH_REDIS_TOKEN
```
Comment (balazsorban44) [inline docs/docs/adapters/upstash-redis.md:24]:
```suggestion
  url: process.env.UPSTASH_REDIS_URL,
  token: process.env.UPSTASH_REDIS_TOKEN
```

**Review by balazsorban44 — APPROVED** [review]
Thanks!

**Pull request merged**